### ISSUE TITLE
ert.saveReusult() and ert.coverageDC()

### ISSUE THREAD

**Issue by stefanazzz** [gimli-org/gimli#265]
### Problem description
Hi,

I used both commands ´ert.saveReusult()` and `ert.coverageDC()` successfully in old environment. 
Updating pygimly and pybert results in error 
"ERTmanager has no attribute ... "
I wonder if these attributes have been removed from ERT between the 2 versions, or is it a mistake I did in installing?

Thanks,
Stefan

### Your environment

**Operating system**:LINUX
**Python version**: 3.7.8 | packaged by conda-forge | (default, Jul 31 2020, 02:25:08)
**pyGIMLi version**: 1.1.0+0.g47c8d810
**Way of installation**:packaged by conda-forge

### Steps to reproduce
```Python
import numpy as np
import pybert as pb
import pygimli as pg
import pygimli.meshtools as mt  # save space
import pandas as pd
import matplotlib
%matplotlib notebook
from matplotlib import pyplot as plt

# ... code to read datafile and perform inversion with pb.ERTManager and ert.invert(..). This part works fine

pybert_version, pygimli_version = ('2.2.11+0.gc93d39b.dirty', '1.0.12+0.g9962fe88.dirty')
ert.saveResult("./")
ert.coverageDC()
--> No problem

pybert_version, pygimli_version = ('2.3.0+0.g9dafa94.dirty', '1.1.0+0.g47c8d810.dirty')
ert.saveResult"./"

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-12-296bee3ae868> in <module>
----> 1 ert.saveResult("./");

AttributeError: 'ERTManager' object has no attribute 'saveResult'


ert.coverageDC()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-23-17045ac1a982> in <module>
      5 import matplotlib.tri as mtri
      6 
----> 7 in2= ert.coverageDC();
      8 #in2=(-min(in2)+in2/(max(in2)-min(in2)))
      9 #in2=in2

AttributeError: 'ERTManager' object has no attribute 'coverageDC'
Tell us how to reproduce this issue. Ideally, you could paste the code that produces the error:
```

```python
import pygimli as pg
...
```

### Expected behavior

To save results in ERTmanager folder with command ert.saveResult().
To read coverage with command ert.coverageDC()

> If possible, please add one or more labels to your issue, e.g. if you expect that your issue is rather a question than a problem with the code, please add the label "question".

**Comment by halbmy** [comment]
A few things have changed recently, `saveResult` was re-added and the coverage is only `ert.coverage()`.
Can you please update to the last version 1.1.1 and see whether the problem remains?

**Comment by stefanazzz** [comment]
Thanks Thomas,
I will do
Stefan

**Comment by stefanazzz** [comment]
Hi, 
I tested 1.1.1 and that solved the problems as suggested. Thanks!
However a different issue has appeared with ert.inv.response() in version 1.1.1:

ert.inv.response()
...
TypeError: 'RVector' object is not callable

Maybe I should close the 265 issue and open a new issue for ert.inv.response()?

**Comment by carsten-forty2** [comment]
please try `ert.inv.response` (without the brackets) as the response vector of the last run.

**Comment by stefanazzz** [comment]
thanks!

**Comment by halbmy** [comment]
Seems all had been solved with 1.1.1 so that we can close the issue. Thank's for reporting.
